source/fab/steps/psyclone.py

```diff
@@ -57,6 +57,7 @@ class MpPayload:
     """
     analysed_x90: Dict[Path, AnalysedX90]
     all_kernel_hashes: Dict[str, int]
+    override_files: List[str]  # filenames (not paths) of hand crafted overrides
```

Comment:
The Path object can hold relative paths. Is that what is needed here or do you need just the name of the file?

Reply:
We just need the name of the file.

Currently, we find an override by comparing just the file names, then construct the override path as `override_folder / filename`, which assumes the override folder is flat.